### ISSUE TITLE
search_issues method ignores expand

### ISSUE THREAD

**Issue by ofajardo** [pycontribs/jira#547]
When using the search_issues method and giving and expand for example 
```python
issues = jac.search_issues(search_string, maxResults=False, expand="renderedField")
```
I do not get a field renderedFields. The same happens with attachments.
Doing it from the issue method works as expected.
```python
curissue = jac.issue(curkey, expand='renderedFields')
```

**Comment by ssbarnea** [comment]
Sounds like a legit bug and one for which we should also add a test case, one that compares fields available on results from `search_issues` and `issue()` when they are called with the same expand parameters. PR are welcomed.

**Comment by chevell** [comment]
@ofajardo I can't reproduce this (note: using jira 2.0rc2):

```
>>> issues = jira.search_issues(jql, expand=['renderedFields'])
>>> issue = issues.pop()
>>> hasattr(issue, 'renderedFields')
True
```

Do you not see the property at all (note it's a direct property of `issue`, not `issue.fields`)? Or does it not contain data you're expecting?

**Comment by ofajardo** [comment]
@dchevell 

Hi thanks a lot for the tip. I updated to 2.0.0 and what you suggest works. 

Now, this does not work for you, does it work in your hands?:

 ```
>>> issues = jac.search_issues("issue=XXX-603", maxResults=False, expand="attachment")
>>> issue = issues[0]
>>> hasattr(issue, 'attachment')
False
>>> hasattr(issue.fields, 'attachment')
False
```

However this works:

```
issue = jac.issue('MDH-603', expand="attachment")
>>> hasattr(issue, 'attachment')
False
>>> hasattr(issue.fields, 'attachment')
True
```

By the way, it is also confusing that in the case of renderedFields it is a member of issue, and in the case of attachment it is a member of issue.fields.

Another thing is I tried to install directly from the repo via setup.py install but I could not manage to get that to work (error: 
Exception: Versioning for this project requires either an sdist tarball, or acce                                                                                                   ss to an upstream git repository. It's also possible that there is a mismatch be                                                                                                   tween the package name in setup.cfg and the argument given to pbr.version.Versio                                                                                                   nInfo. Project name jira was given, but was not able to be found.)
